<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MEditImgItem</class>
 <widget class="QDialog" name="MEditImgItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frCol">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;name</string>
          </property>
          <property name="buddy">
           <cstring>txtColName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtColName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;directory</string>
          </property>
          <property name="buddy">
           <cstring>txtColDir</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtColDir"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frBook">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>n&amp;ame</string>
          </property>
          <property name="buddy">
           <cstring>txtBookName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtBookName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>file na&amp;me</string>
          </property>
          <property name="buddy">
           <cstring>txtBookFile</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtBookFile"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpCoords">
        <property name="title">
         <string>area and time period</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string notr="true">&amp;long 1</string>
               </property>
               <property name="buddy">
                <cstring>spnLong1</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spnLong1">
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string notr="true">l&amp;ong 2</string>
               </property>
               <property name="buddy">
                <cstring>spnLong2</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QDoubleSpinBox" name="spnLong2">
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string notr="true">la&amp;t 1</string>
               </property>
               <property name="buddy">
                <cstring>spnLat1</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spnLat1">
               <property name="minimum">
                <double>-90.000000000000000</double>
               </property>
               <property name="maximum">
                <double>90.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string notr="true">lat &amp;2</string>
               </property>
               <property name="buddy">
                <cstring>spnLat2</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDoubleSpinBox" name="spnLat2">
               <property name="minimum">
                <double>-90.000000000000000</double>
               </property>
               <property name="maximum">
                <double>90.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="tbFromClipboard">
             <property name="text">
              <string>from
clipboard</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>f&amp;rom year</string>
             </property>
             <property name="buddy">
              <cstring>spnFromTime</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnFromTime">
             <property name="minimum">
              <number>-9900</number>
             </property>
             <property name="maximum">
              <number>9900</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>to &amp;year</string>
             </property>
             <property name="buddy">
              <cstring>spnToTime</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnToTime">
             <property name="minimum">
              <number>-9900</number>
             </property>
             <property name="maximum">
              <number>9900</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MEditImgItem</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MEditImgItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
